storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -2306,10 +2306,19 @@ static bool innodb_init(bool init_dd, bool for_apply_log) {
   }
 
   lsn_t to_lsn = LSN_MAX;
+
   if (for_apply_log && (metadata_type == METADATA_FULL_BACKUP ||
                         xtrabackup_incremental_dir != nullptr)) {
-    to_lsn = (xtrabackup_incremental_dir == nullptr) ? metadata_last_lsn
-                                                     : incremental_last_lsn;
+    /* backup taken prior to 8.0.23 can have checkoint lsn more than last
+      lsn. if so use checkpoint_lsn instead of last_lsn */
+    if (xtrabackup_incremental_dir == nullptr) {
```

Comment:
```suggestion
    /* Backups taken prior to 8.0.23 can have checkoint lsn more than last
    copied lsn. If so, use checkpoint_lsn instead of last_lsn */
    if (xtrabackup_incremental_dir == nullptr) {
```

Reply:
This review comment is not addressed (may be lost). Please address